test/Bridges/Constraint/square.jl

```diff
@@ -240,6 +240,80 @@ function test_square_warning()
     return
 end
 
+function test_VectorNonlinearFunction_symmetric()
+    inner = MOI.Utilities.Model{Float64}()
+    model = MOI.Bridges.Constraint.Square{Float64}(inner)
+    x = MOI.add_variables(model, 3)
+    fis = Any[MOI.ScalarNonlinearFunction(:log, Any[x[i]]) for i in 1:3]
+    f = MOI.VectorNonlinearFunction(Any[fis[1], fis[2], fis[2], fis[3]])
+    c = MOI.add_constraint(model, f, MOI.PositiveSemidefiniteConeSquare(2))
```

Comment:
@blegat this is now working.

But if you print `model` you'll get a horrible function, because VectorNonlinear doesn't simplify `(f(x) - g(x)) + g(x) --> f(x)` at the expression level.